trp_use_case/model/use_case.py

```diff
@@ -81,15 +82,15 @@ def _get_number(self):
         """
         self.env.cr.execute(
             """SELECT
-                id,
-                (
-                    SELECT COUNT(*)
-                    FROM use_case
-                    WHERE
-                        collection_id = uc.collection_id
-                        AND active = true
-                        AND sequence <= uc.sequence
-                )
+            id,
+            (
+            SELECT COUNT(*)
+            FROM use_case
+            WHERE
+            collection_id = uc.collection_id
+            AND active = true
+            AND sequence <= uc.sequence
+        )
```

Comment:
the original indentation was meant to increase readability, to you think it's more readable this way?